boms/cloud-lts-bom/pom.xml

```diff
@@ -0,0 +1,302 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>gcp-lts-bom</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Google Cloud Long Term Support BOM</name>
```

Comment:
To avoid premature release, can we remove all this metadata for the time being? That will enable us to test tools but prevent us from releasing to Sonatype.

Reply:
Here are reasons why accidental release does not happen:

- The BOM release process happens in boms/cloud-oss-bom directory ([source](https://source.corp.google.com/piper///depot/google3/cloud/java/tools/cloud-opensource-java/bom/release_build.sh;l=8))
- groupId `com.google.cloud` is protected. Only selected (non-personal) account can upload artifacts.
- Maven Central blocks the `-SNAPSHOT` version.
- Our release process have human's eyes in releasing artifacts via Sonatype staging.

If you strongly want to remove the metadata from this draft, then I will have a copy of this file in my local to test things. At that time, I would carry the risk of releasing premature file from my local. (Then I would prefer to keep this PR open)